markdown-mode.el

```diff
@@ -9087,7 +9087,7 @@ This function assumes point is on a table."
             (indent (progn (looking-at "[ \t]*") (match-string 0)))
             ;; Split table in lines and save column format specifier
             (lines (mapcar (lambda (l)
-                             (if (string-match-p "\\`[ \t]*|[-:]" l)
+                             (if (string-match-p "\\`[ \t]*| *[-:]" l)
```

Comment:
Is tab not allowed ?

```suggestion
                             (if (string-match-p "\\`[ \t]*|[ \t]*[-:]" l)
```

Reply:
I don't think tabs make much sense here, but allowing tabs does no harm.